src/test/java/co/rsk/bitcoinj/core/BtcTransactionTest.java

```diff
@@ -389,4 +390,126 @@ public void testSerializerTxtWithWitness() {
+        BtcTransaction tx = new BtcTransaction(mainnet, rawTx);
+
+        Script script = new Script(
```

Comment:
This script is repeated three times. Let's move the rawTxInHex out of the test. It could be a constant. Wdyt?